src/screens/Home.js

```diff
@@ -27,20 +26,28 @@ import ModeIcon from '../components/ToolButton/assets/mode.svg';
 export default ({
   //nav provider
   navigation,
+
   //state props
   categories,
+  learntPhrases,
   nativeLanguage,
+
   //actions
+  setPhrases,
   setCategories,
+  addLearntPhrase,
   setCurrentCategory,
-  setPhrases,
 }) => {
   useEffect(() => {
     // fetch categories
     const categories = getAllCategories();
     setCategories(categories);
   }, []);
 
+  useEffect(() => {
+    addLearntPhrase(learntPhrases);
```

Comment:
learntPhrases will always be empty at the beginning so I don't see how it would work. You should have something like synchronize. (this should be already available from Clopedias changes)